src/main/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceImpl.java

```diff
@@ -231,4 +234,13 @@ public PrivilegeSummary getLatestBookingIEPSummaryForOffender(final String offen
         Optional.ofNullable(booking).orElseThrow(EntityNotFoundException.withId(offenderNo));
         return bookingService.getBookingIEPSummary(booking.getBookingId(), withDetails);
     }
+
+    @Override
+    public OffenderDamageObligationResponse getOffenderDamageObligations(final String offenderNo, final String status) {
```

Comment:
no `VerifyOffenderAccess`?

Reply:
I've placed it on the service. Would you prefer it on the Resource instead?

Reply:
just spotted it lower down :grinning: